### PR TITLE
Add and use Jenkinsfile

### PR DESCRIPTION
**Pull request by stephendonner**
Addresses https://github.com/mozilla/snippets-tests/issues/68

@davehunt @glogiotatidis r?

Ad-hoc run here, passing: http://webqa-ci-staging1.qa.scl3.mozilla.com:8080/job/snippets.pipeline/16/console

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,9 +0,0 @@
-[flake8]
-ignore=E501
-
-[tool:pytest]
-python_files=test_*.py
-addopts=-n=auto --verbose -r=a --junit-xml=results/junit.xml
-testpaths=tests
-xfail_strict=true
-base_url=https://snippets.allizom.org
```
Comment (davehunt) [inline setup.cfg]: We shouldn't remove this section entirely - did you mean to move it into `tox.ini`?

Comment (stephendonner) [inline setup.cfg]: Sorry; yes, I did - done, now!

**Comment by stephendonner** [comment]
@davehunt final r?  thx!

File: setup.cfg (hunk 1 above)
Comment (stephendonner) [inline setup.cfg]: Yes, good point - @glogiotatidis would you be supportive/interested in that?

**Pull request merged**